utils/wtmcu/Makefile

```diff
@@ -0,0 +1,56 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wtmcu
+PKG_VERSION:=0.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://gitlab.com/evgenyz/$(PKG_NAME)/-/archive/v$(PKG_VERSION)
+PKG_HASH:=557f8b6a9771dcee094692da4480274695ec2af53871a3ddea069ee94e207509
+
+PKG_LICENSE:=BSD-2-Clause
+PKG_MAINTAINER:=Evgeny Kolesnikov <evgenyz@gmail.com>
+
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-v$(PKG_VERSION)
+
+CMAKE_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/wtmcu
+	SECTION:=utils
+	CATEGORY:=Utilities
+	TITLE:=Weltrend MCU control daemon and console tool
+	MENU:=1
+	ABI_VERSION:=$(PKG_VERSION)
+	DEPENDS:=+libubox +libubus
+endef
+
+define Package/wtmcu/description
+	This package provides facilities to control Weltrend MCU.
+endef
+
+define Package/wtmcu/conffiles
+	/etc/config/wtmcu
+endef
+
+TARGET_CFLAGS+=-std=gnu99
```

Comment:
This should not be needed.

Reply:
This may be needed depending on version of compiler

Reply:
What I mean is that the package should be compatible with all versions. If not, this is a bug that should be fixed.

Reply:
Well, current toolchain used for mvebu target does indeed defaults to C standard which is fresh enough to drop this flag. But, irrelevant to this package, what do you mean by "compatible with all versions"? This flag just asks for a particular C standard revision. If gcc does not support it, the flag would do nothing, it won't compile. Are you saying that package should only be compatible with what is default for currently used gcc version?

Reply:
@neheb 
No it's not, default value may (will) change depending on GCC/compiler version. Blantantly ignoring this fact will however introduce breakage and possibly even bugs.